libc/include/llvm-libc-types/rsize_t.h

```diff
@@ -0,0 +1,16 @@
+//===-- Definition of rsize_t type ----------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_TYPES_RSIZE_T_H
+#define LLVM_LIBC_TYPES_RSIZE_T_H
+
+#include <stddef.h>
```

Comment:
```suggestion
#include "size_t.h"
```

Reply:
Don't forget to make this change, please.

Reply:
Should this change also be made in `hdr/`?